flutter/android/android.mk

```diff
@@ -44,6 +45,37 @@ flutter/android/libs/build:
 		${backend_samsung_android_target} \
 		//flutter/cpp/flutter:libbackendbridge.so
 
+.PHONY: cmdline/android/bins/build
+cmdline/android/bins/build:
```

Comment:
Can you explain a bit when to use this make target?

Reply:
This is for building the main.cc along with the backend and generate a package.

Reply:
Is it for local testing? Would it be better to locate it inside `mobile_back_qti` to keep the common code base compact? Similar to this Makefile, which is mainly used for local testing and development:
https://github.com/mlcommons/mobile_app_open/blob/85d380f7f1128712e87c0e6f5cb3e19e1ac06551/mobile_back_apple/dev-utils/Makefile#L22-L24

Reply:
Actually this commandline tool could be used for other backends as well. 
The steps to build are similar to the way we build the flutter app... with similar options to be passed.

Reply:
Ok. Can you create a new `Makefile` in `flutter/cpp/binary/` and put all the `*cmdline*` targets there? Since the `main.cc` is not only for Android.

Reply:
main.cc is not just for android, but the binary being built using this target is for android.
similar steps are there for windows as well.

Reply:
The idea was to keep the Makefile target inside the `android` folder to build Android app, the target inside `ios` folder to build iOS app, and so on.
Therefore, I would like to keep the targets to build the `main.cc` inside `flutter/cpp/binary/` folder.

Reply:
You can create a new file `flutter/cpp/binary/main.mk` and put the `*cmdline*` targets there.
Then in the `flutter/flutter.mk` file, you can add a new line to point to it.
```
include flutter/cpp/binary/main.mk
```

Reply:
sure... i will check this out.

Reply:
I have pushed this change... Can you please review?